### PR TITLE
Samsung Internet, Updating H-N files, in api/

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
A checklist to help your pull request get merged faster:
- [ ] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

### CODE REVIEW

File: api/HTMLCanvasElement.json [hunk 1]
```diff
@@ -356,9 +356,15 @@
               "safari_ios": {
                 "version_added": null
               },
-              "samsunginternet_android": {
-                "version_added": null
-              },
+              "samsunginternet_android": [
+                {
+                  "version_added": "3.0"
```
Comment (vinyldarkscratch) [inline api/HTMLCanvasElement.json]:
```suggestion
                  "version_added": "2.0"
```

File: api/HTMLCanvasElement.json [hunk 2]
```diff
@@ -914,7 +920,7 @@
               "version_added": "8"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/HTMLCanvasElement.json]:
```suggestion
              "version_added": "1.5"
```

File: api/HTMLCanvasElement.json [hunk 3]
```diff
@@ -963,7 +969,7 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLCanvasElement.json]:
```suggestion
              "version_added": "1.5"
```

File: api/HTMLCanvasElement.json [hunk 4]
```diff
@@ -1012,7 +1018,7 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLCanvasElement.json]:
```suggestion
              "version_added": "1.5"
```

File: api/HTMLContentElement.json [hunk 5]
```diff
@@ -51,7 +51,7 @@
             "version_added": null
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/HTMLContentElement.json]:
```suggestion
            "version_added": "3.0"
```

File: api/HTMLContentElement.json [hunk 6]
```diff
@@ -114,7 +114,7 @@
               "version_added": null
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/HTMLContentElement.json]:
```suggestion
              "version_added": "3.0"
```

File: api/HTMLContentElement.json [hunk 7]
```diff
@@ -178,7 +178,7 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLContentElement.json]:
```suggestion
              "version_added": "3.0"
```

File: api/HTMLImageElement.json [hunk 8]
```diff
@@ -1104,6 +1173,9 @@
             "safari_ios": {
               "version_added": "8"
             },
+            "samsunginternet_android": {
+              "version_added": "3.0"
```
Comment (vinyldarkscratch) [inline api/HTMLImageElement.json]:
```suggestion
              "version_added": "2.0"
```

File: api/HTMLPictureElement.json [hunk 9]
```diff
@@ -61,7 +61,7 @@
             "version_added": "9.3"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/HTMLPictureElement.json]:
```suggestion
            "version_added": "3.0"
```

File: api/HTMLShadowElement.json [hunk 10]
```diff
@@ -51,7 +51,7 @@
             "version_added": false
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/HTMLShadowElement.json]:
```suggestion
            "version_added": "3.0"
```

File: api/HTMLShadowElement.json [hunk 11]
```diff
@@ -98,7 +98,7 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/HTMLShadowElement.json]:
```suggestion
              "version_added": "3.0"
```

File: api/HTMLTemplateElement.json [hunk 12]
```diff
@@ -35,7 +35,7 @@
             "version_added": "8"
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/HTMLTemplateElement.json]:
```suggestion
            "version_added": "1.5"
```

File: api/HTMLTemplateElement.json [hunk 13]
```diff
@@ -82,7 +82,7 @@
               "version_added": "8"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/HTMLTemplateElement.json]:
```suggestion
              "version_added": "1.5"
```

File: api/IDBCursor.json [hunk 14]
```diff
@@ -72,12 +72,12 @@
           },
           "samsunginternet_android": [
             {
-              "version_added": true
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/IDBCursor.json]:
```suggestion
              "version_added": "1.5"
```

File: api/IDBCursor.json [hunk 15]
```diff
@@ -72,12 +72,12 @@
...
             },
             {
               "version_removed": "7.0",
               "prefix": "webkit",
-              "version_added": true
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/IDBCursor.json]:
```suggestion
              "version_added": "1.5"
```

File: api/IDBCursor.json [hunk 16]
```diff
@@ -768,12 +768,12 @@
             },
             "samsunginternet_android": [
               {
-                "version_added": true
+                "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/IDBCursor.json]:
```suggestion
                "version_added": "1.5"
```

File: api/IDBCursor.json [hunk 17]
```diff
@@ -768,12 +768,12 @@
...
               },
               {
                 "version_removed": "7.0",
                 "prefix": "webkit",
-                "version_added": true
+                "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/IDBCursor.json]:
```suggestion
                "version_added": "1.5"
```

File: api/IDBCursorWithValue.json [hunk 18]
```diff
@@ -58,12 +58,12 @@
           },
           "samsunginternet_android": [
             {
-              "version_added": true
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/IDBCursorWithValue.json]:
```suggestion
              "version_added": "1.5"
```

File: api/IDBCursorWithValue.json [hunk 19]
```diff
@@ -58,12 +58,12 @@
...
             },
             {
               "version_removed": "7.0",
               "prefix": "webkit",
-              "version_added": true
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/IDBCursorWithValue.json]:
```suggestion
              "version_added": "1.5"
```

**Comment by AdaRoseCannon** [comment]
These look fine, I wonder why my script was mislabelling these.

File: api/HTMLElement.json [hunk 20]
```diff
@@ -1298,6 +1358,10 @@
             "safari_ios": {
               "version_added": null
             },
+            "samsunginternet_android": {
+              "version_added": "1.0",
+              "version_removed": "2.0"
```
Comment (vinyldarkscratch) [inline api/HTMLElement.json]:
```suggestion
              "version_removed": "1.5"
```

File: api/HTMLElement.json [hunk 21]
```diff
@@ -1346,6 +1410,10 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLElement.json]:
```suggestion
              "version_removed": "1.5"
```

File: api/HTMLElement.json [hunk 22]
```diff
@@ -1394,6 +1462,10 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLElement.json]:
```suggestion
              "version_removed": "1.5"
```

File: api/HTMLElement.json [hunk 23]
```diff
@@ -1442,6 +1514,10 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLElement.json]:
```suggestion
              "version_removed": "1.5"
```

File: api/HTMLElement.json [hunk 24]
```diff
@@ -1490,6 +1566,10 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLElement.json]:
```suggestion
              "version_removed": "1.5"
```

File: api/HTMLElement.json [hunk 25]
```diff
@@ -1538,6 +1618,10 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLElement.json]:
```suggestion
              "version_removed": "1.5"
```

File: api/HTMLElement.json [hunk 26]
```diff
@@ -3231,6 +3366,9 @@
             "safari_ios": {
               "version_added": true
             },
+            "samsunginternet_android": {
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/HTMLElement.json]:
```suggestion
              "version_added": "1.5"
```

File: api/HTMLTrackElement.json [hunk 27]
```diff
@@ -62,6 +62,9 @@
           "safari_ios": {
             "version_added": true
           },
+          "samsunginternet_android": {
+            "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/HTMLTrackElement.json]:
```suggestion
            "version_added": "1.5"
```

File: api/HTMLTrackElement.json [hunk 28]
```diff
@@ -178,6 +184,9 @@
             "safari_ios": {
               "version_added": true
             },
+            "samsunginternet_android": {
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/HTMLTrackElement.json]:
```suggestion
              "version_added": "1.5"
```

File: api/HTMLTrackElement.json [hunk 29]
```diff
@@ -249,6 +258,9 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLTrackElement.json]:
```suggestion
              "version_added": "1.5"
```

File: api/HTMLTrackElement.json [hunk 30]
```diff
@@ -320,6 +332,9 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLTrackElement.json]:
```suggestion
              "version_added": "1.5"
```

File: api/HTMLTrackElement.json [hunk 31]
```diff
@@ -391,6 +406,9 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLTrackElement.json]:
```suggestion
              "version_added": "1.5"
```

File: api/HTMLTrackElement.json [hunk 32]
```diff
@@ -464,6 +482,9 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLTrackElement.json]:
```suggestion
              "version_added": "1.5"
```

File: api/HTMLTrackElement.json [hunk 33]
```diff
@@ -535,6 +556,9 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLTrackElement.json]:
```suggestion
              "version_added": "1.5"
```

File: api/HTMLTrackElement.json [hunk 34]
```diff
@@ -606,6 +630,9 @@
...
```
Comment (vinyldarkscratch) [inline api/HTMLTrackElement.json]:
```suggestion
              "version_added": "1.5"
```

File: api/KeyboardEvent.json [hunk 35]
```diff
@@ -563,7 +563,7 @@
               "version_added": "10.3"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "3.0"
```
Comment (vinyldarkscratch) [inline api/KeyboardEvent.json]:
```suggestion
              "version_added": "2.0"
```

File: api/KeyboardEvent.json [hunk 36]
```diff
@@ -1678,7 +1678,8 @@
               "version_added": "5.1"
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/KeyboardEvent.json]:
```suggestion
              "version_added": "1.5",
```

File: api/MutationObserver.json [hunk 37]
```diff
@@ -62,9 +62,16 @@
               "prefix": "WebKit"
             }
           ],
-          "samsunginternet_android": {
-            "version_added": true
-          },
+          "samsunginternet_android": [
+            {
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/MutationObserver.json]:
```suggestion
              "version_added": "1.5"
```

File: api/MutationObserver.json [hunk 38]
```diff
@@ -62,9 +62,16 @@
...
+            },
+            {
+              "prefix": "WebKit",
+              "version_added": "1.0",
+              "version_removed": "2.0"
```
Comment (vinyldarkscratch) [inline api/MutationObserver.json]:
```suggestion
              "version_removed": "1.5"
```

File: api/MutationObserver.json [hunk 39]
```diff
@@ -145,9 +152,16 @@
                 "prefix": "WebKit"
               }
             ],
-            "samsunginternet_android": {
-              "version_added": true
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/MutationObserver.json]:
```suggestion
                "version_added": "1.5"
```

File: api/MutationObserver.json [hunk 40]
```diff
@@ -145,9 +152,16 @@
...
+              },
+              {
+                "prefix": "WebKit",
+                "version_added": "1.0",
+                "version_removed": "2.0"
```
Comment (vinyldarkscratch) [inline api/MutationObserver.json]:
```suggestion
                "version_removed": "1.5"
```

File: api/Navigator.json [hunk 41]
```diff
@@ -459,7 +460,7 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/Navigator.json]:
```suggestion
              "version_added": "3.0"
```

File: api/Navigator.json [hunk 42]
```diff
@@ -892,9 +893,15 @@
             "safari_ios": {
               "version_added": false
             },
-            "samsunginternet_android": {
-              "version_added": true
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/Navigator.json]:
```suggestion
                "version_added": "3.0"
```

File: api/Navigator.json [hunk 43]
```diff
@@ -892,9 +893,15 @@
...
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/Navigator.json]:
```suggestion
                "version_added": "1.5"
```

File: api/Navigator.json [hunk 44]
```diff
@@ -974,9 +981,16 @@
...
-            "samsunginternet_android": {
-              "version_added": false
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "6.0"
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Navigator.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Navigator.json [hunk 45]
```diff
@@ -1209,7 +1224,7 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/Navigator.json]:
```suggestion
              "version_added": "3.0"
```

File: api/Navigator.json [hunk 46]
```diff
@@ -2203,7 +2224,11 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "3.0",
```
Comment (vinyldarkscratch) [inline api/Navigator.json]:
```suggestion
              "version_added": "2.0",
```

File: api/NavigatorConcurrentHardware.json [hunk 47]
```diff
@@ -35,7 +35,7 @@
             "version_added": null
           },
           "samsunginternet_android": {
-            "version_added": true
+            "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/NavigatorConcurrentHardware.json]:
```suggestion
            "version_added": "3.0"
```

File: api/NavigatorConcurrentHardware.json [hunk 48]
```diff
@@ -82,7 +82,7 @@
               "version_added": null
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/NavigatorConcurrentHardware.json]:
```suggestion
              "version_added": "3.0"
```

File: api/NavigatorConcurrentHardware.json [hunk 49]
```diff
@@ -130,7 +130,7 @@
...
```
Comment (vinyldarkscratch) [inline api/NavigatorConcurrentHardware.json]:
```suggestion
              "version_added": "3.0"
```

File: api/NavigatorLanguage.json [hunk 50]
```diff
@@ -139,6 +146,10 @@
             "safari_ios": {
               "version_added": true
             },
+            "samsunginternet_android": {
+              "version_added": "3.0",
```
Comment (vinyldarkscratch) [inline api/NavigatorLanguage.json]:
```suggestion
              "version_added": "2.0",
```

File: api/NavigatorOnLine.json [hunk 51]
```diff
@@ -134,6 +140,9 @@
             "safari_ios": {
               "version_added": null
             },
+            "samsunginternet_android": {
+              "version_added": "3.0"
```
Comment (vinyldarkscratch) [inline api/NavigatorOnLine.json]:
```suggestion
              "version_added": "2.0"
```

File: api/NetworkInformation.json [hunk 52]
```diff
@@ -34,6 +34,9 @@
           "safari_ios": {
             "version_added": false
           },
+          "samsunginternet_android": {
+            "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/NetworkInformation.json]:
```suggestion
            "version_added": "3.0"
```

File: api/NetworkInformation.json [hunk 53]
```diff
@@ -78,6 +81,9 @@
             "safari_ios": {
               "version_added": false
             },
+            "samsunginternet_android": {
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/NetworkInformation.json]:
```suggestion
              "version_added": "3.0"
```

File: api/NetworkInformation.json [hunk 54]
```diff
@@ -124,6 +130,9 @@
...
```
Comment (vinyldarkscratch) [inline api/NetworkInformation.json]:
```suggestion
              "version_added": "3.0"
```

File: api/NetworkInformation.json [hunk 55]
```diff
@@ -169,6 +178,9 @@
...
```
Comment (vinyldarkscratch) [inline api/NetworkInformation.json]:
```suggestion
              "version_added": "3.0"
```

File: api/NetworkInformation.json [hunk 56]
```diff
@@ -215,6 +227,9 @@
...
```
Comment (vinyldarkscratch) [inline api/NetworkInformation.json]:
```suggestion
              "version_added": "3.0"
```

File: api/NetworkInformation.json [hunk 57]
```diff
@@ -305,6 +323,9 @@
...
```
Comment (vinyldarkscratch) [inline api/NetworkInformation.json]:
```suggestion
              "version_added": "3.0"
```

File: api/NetworkInformation.json [hunk 58]
```diff
@@ -396,6 +420,9 @@
...
```
Comment (vinyldarkscratch) [inline api/NetworkInformation.json]:
```suggestion
              "version_added": "3.0"
```

File: api/NetworkInformation.json [hunk 59]
```diff
@@ -441,6 +468,9 @@
...
```
Comment (vinyldarkscratch) [inline api/NetworkInformation.json]:
```suggestion
              "version_added": "3.0"
```

**Comment by AdaRoseCannon** [comment]
This failed because the build timed out

**Comment by ddbeck** [comment]
I restarted the build and it has now passed. 👍

**Comment by vinyldarkscratch** [comment]
Thanks, @AdaRoseCannon!  We've just got this last part, and then we're ready:

> It looks like in most files, anywhere it says "2.0" should actually be "1.5", and "3.0" should be "2.0". Maybe we have a mismatch of what Samsung Internet versions have which Chrome versions?
> 
> Could we update those version numbers accordingly in the following files?
> 
> * IDBDatabase
> * IDBEnvironment
> * IDBFactory
> * IDBIndex
> * IDBObjectStore
> * IDBOpenDBRequest
> * IDBRequest
> * IDBTransaction
> * IDBVersionChangeEvent
> * MediaSource
> * MediaStream
> * MutationObserverInit
> 
> Once they're all done, then we should be good to go for merging!

**Pull request merged**